patterns/1-initial/extensions-to-manage-contributions-at-scale.md

```diff
@@ -0,0 +1,81 @@
+## Title
+
+Extensions to Manage Contributions at Scale
+
+## Patlet
+
+TBD
```

Comment:
```suggestion
An InnerSource project is receiving too many contributions for the maintainers to handle, leading to low quality code reviews and premature rejection of feature requests.
By offering an extension mechanism outside of the core project, the maintainers enable contributors to experiment with their contributed extensions faster, while also clearly indicating to project users what is a core feature and what an experimental extension.
```

I tried myself on a Patlet.
It isn't easy to compress this pattern into a short Patlet, as the pattern contains so much. but I tried :)